src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -24,11 +24,13 @@ import { IGeoConfig, IGeoData } from "../../../interfaces/GeoChart";
 import "../../../../styles/scss/geoChart.scss";
 import { handlePushpinMouseEnter, handlePushpinMouseLeave } from "./geoChartTooltip";
 import { getGeoData } from "../../../helpers/geoChart";
+import { IVisualizationProperties } from "../../../internal/interfaces/Visualization";
 
 export interface IGeoChartRendererProps {
     config: IGeoConfig;
     execution: Execution.IExecutionResponses;
     afterRender(): void;
+    pushData(data: any): void;
```

Comment:
we already have interface for payload. It would be nice to have documented all possible payloads of this callback 
https://github.com/gooddata/gooddata-react-components/blob/develop/src/interfaces/PushData.ts#L18